src/Windowing/Silk.NET.Windowing.Common/Internals/ViewImplementationBase.cs

```diff
@@ -0,0 +1,259 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+using System.Buffers;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Drawing;
+using System.Threading;
+using Silk.NET.Core.Contexts;
+
+// No immediate urgency to document these as they're not intended for public consumption.
+#pragma warning disable 1591
+
+namespace Silk.NET.Windowing.Internals
+{
+    /// <summary>
+    /// Abstracts away common view functions to ease implementation of the windowing API.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public abstract class ViewImplementationBase : IView
```

Comment:
This shouldn't be public imo

Reply:
I originally thought that too, however I figured it could be helpful if ever someone makes a third-party windowing backend? Can make internal if you think otherwise as I do agree some users may get confused if porting over from OpenTK.

Reply:
Yeah I would just make it internal, if someone ever needs the source, which imo is unlikely, the license is permissive enough for them to just copy it.

Reply:
What namespace should WindowPlatformAttribute go under? That must remain public but again not intended for common consumption.

Reply:
I'd just put it into the normal Windowing namespace, I mean it's not like the user can come complain if they use an Attribute for no reason.

Reply:
Done